boards/circuit_playground_express/Cargo.toml

```diff
@@ -1,31 +1,36 @@
 [package]
 name = "circuit_playground_express"
-version = "0.11.1"
+categories = ["embedded", "hardware-support", "no-std"]
+version = "0.12.0"
```

Comment:
Please leave the crate version as-is. Our versions are automatically updated by CI upon release :)